vnext/rn-cli.config.js

```diff
@@ -56,5 +56,10 @@ if (fs.existsSync(path.resolve(__dirname, '../../scripts/metro-resources.js')))
 config.resolver.extraNodeModules['SnapshotViewIOS'] = path.resolve(__dirname, 'Libraries/RCTTest/SnapshotViewIOS');
 config.resolver.hasteImplModulePath = path.resolve(__dirname, 'jest/hasteImpl.js');
 
+// Check that we have built our JS files before running the bundler, otherwise we'll get a harder to diagnose "Unable to resolve module" error
+if (!fs.existsSync(path.resolve(__dirname, 'lib/Libraries/Components/AccessibilityInfo/AccessibilityInfo.uwp.js'))) {
```

Comment:
Is there something special about this file versus some other file?

Reply:
No I could have picked any of the files that are produced from build.

Reply:
I'm not sure if you can tell whether a build has been done by whether a directory exists or having anything named `.js` or `uwp.js`. This is just fragile if this particular file ever is changed.

Reply:
Its true that this file could stop being produced, but its something that would be found immediately, since then you wouldn't be able to bundle anything.  I don't think its worth writing code to troll through the whole directory looking looking for any .js file.

I also expect that in the future there will be other processes than just tsc that will put js files in the lib folder. -- In particular the change to support 0.61 when they get rid of hastemaps will involve other js files being in the lib folder.

Reply:
Fine.